### ISSUE TITLE
Explore design options for communicating accessibility on DS site components

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4382]
This ticket is a follow on to the explorations done in https://github.com/grommet/hpe-design-system/issues/4261. Building off of the work done in #4261 we should get more into specific design recommendations for how we can communicate accessibility for components on the DS site.

Deliverables:
- [x] Create a design recommendation based on discussions captured here: https://www.figma.com/board/Ix52HUwpfZ8SNDQbp6uyIn/Accessibility-communication-on-the-DS-site?node-id=0-1&node-type=canvas&t=p9jO8dh33fuVoNDm-0
- [x] Ensure design is aligned with success criteria: https://www.figma.com/board/Ix52HUwpfZ8SNDQbp6uyIn/Accessibility-communication-on-the-DS-site?node-id=1-652&node-type=section&t=QMH5tiBLT6s2csa6-0
- [x] Share design explorations with the team (Matt would appreciate dedicated mid-sprint visibility into the work in progress explorations -- outside of show and tell)

**Update**
Lets scope this ticket to the following success criteria (see [Matt's comment here](https://www.figma.com/board/Ix52HUwpfZ8SNDQbp6uyIn?node-id=1-652#1023353749)):

Primary - treat as a report card which communicates the components current state and performance
- A reader should have at-a-glance understanding of the component's accessibility health
- A reader should have at-a-glance understanding of the which WCAG A-level the component meets
- A reader should know what WCAG rules are applicable

Consider all the "education" aspects as potential scope for a future phase.

**Comment by halocline** [comment]
@jcfilben RE: " Create a design recommendation based on discussions captured here:" Do you want a single recommendation? Or would it be helpful to have 2-3 options + recommended option?

**Comment by halocline** [comment]
@jcfilben - See [comment here](https://github.com/grommet/hpe-design-system/issues/4261#issuecomment-2472039803). For the current scope, we should focus on providing visibility and transparency so that DS consumers feel assured that components meet their A11y needs.

**Comment by jcfilben** [comment]
> @jcfilben RE: " Create a design recommendation based on discussions captured here:" Do you want a single recommendation? Or would it be helpful to have 2-3 options + recommended option?

I think it would be helpful to see a few options that the team can give feedback on

> @jcfilben - See https://github.com/grommet/hpe-design-system/issues/4261#issuecomment-2472039803. For the current scope, we should focus on providing visibility and transparency so that DS consumers feel assured that components meet their A11y needs.

Agree, thanks for helping narrow the scope

**Comment by luketa8** [comment]
Added [first mockup](https://www.figma.com/design/SCVuMdA7qzlnbkXd4Pv9ib/%234382---Accessibility-information-on-DS-site?node-id=0-1&node-type=canvas&t=PPYLUfAmKy4yUw66-0). 

We should discuss next which level of detail (total, group or individual) we will go for in the MVP.

**Comment by luketa8** [comment]
[MVP Designs](https://www.figma.com/design/SCVuMdA7qzlnbkXd4Pv9ib/%234382---Communicating-accessibility-on-DS-site?node-id=128-5862&t=vr9Cx8glZ0aNh1JE-4) ready for review/signoff.

**Comment by luketa8** [comment]
Thanks for the comments, Jessica. I added new Tag examples, updated the text with verbiage from W3 site and added several Status variations. I think this Issue is ready to move to Done.

**Comment by luketa8** [comment]
Updated [MVP mockup](https://www.figma.com/design/SCVuMdA7qzlnbkXd4Pv9ib/%234382---Communicating-accessibility-on-DS-site?node-id=197-1933&t=gJOG3WktUNJXyoUf-4) incorporating Matt's comments. 

Let's schedule a meeting to discuss on Monday/Tuesday.

**Comment by luketa8** [comment]
Met with Jessica and Matt. 

[Mockups](https://www.figma.com/design/SCVuMdA7qzlnbkXd4Pv9ib/%234382---Communicating-accessibility-on-DS-site?node-id=197-1933&t=1ukzQiXf3kAjo9Uy-4) Ready for Dev. Closing this issue, thx!
